### PR TITLE
Dead cod

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: src/world/dead/kzn_19/EA8AE0.c [hunk 1]
```diff
@@ -15,59 +15,272 @@ static char* N(exit_str_0) = "kmr_23";
...
+    VINE_3      = 3,
+    NUM_VINES   = 4
+};
+
+enum {
+    VINE_0_BASE  = 0x80200000,
+    VINE_1_BASE  = 0x80204000,
+    VINE_2_BASE  = 0x80207000,
+    VINE_3_BASE  = 0x8020A000,
+}; //TODO shiftaility -- hard-coded addresses in gBackgroundImage
```
Comment (nanaian) [inline src/world/dead/kzn_19/EA8AE0.c:31]:
```suggestion
}; // TODO shiftability -- hard-coded addresses in gBackgroundImage
```

File: src/world/dead/kzn_19/EA8AE0.c [hunk 2]
```diff
@@ -15,59 +15,272 @@ static char* N(exit_str_0) = "kmr_23";
...
+        vine->points[count].x = evt_get_float_variable(dummyEvtPtr, LVar1);
+        vine->points[count].y = evt_get_float_variable(dummyEvtPtr, LVar2);
+        vine->points[count].z = evt_get_float_variable(dummyEvtPtr, LVar3);
+        count++;
+    } while (dummyEvtPtr->varTable[0] != 0);
+    vine->numPoints = count;
+}
 
+// SetVineBonePos
 ApiStatus func_80240BD4_EA96B4(Evt *script, s32 isInitialCall) {
```
Comment (nanaian) [inline src/world/dead/kzn_19/EA8AE0.c:60]: Name this?

File: src/world/dead/kzn_19/EA8AE0.c [hunk 3]
```diff
@@ -15,59 +15,272 @@ static char* N(exit_str_0) = "kmr_23";
...
+
+enum {
+    VINE_0_BASE  = 0x80200000,
+    VINE_1_BASE  = 0x80204000,
+    VINE_2_BASE  = 0x80207000,
+    VINE_3_BASE  = 0x8020A000,
+}; //TODO shiftaility -- hard-coded addresses in gBackgroundImage
+
+// make_vine_interpolation
+void func_80240B00_EA95E0(LavaPiranhaVine* vine) {
```
Comment (nanaian) [inline src/world/dead/kzn_19/EA8AE0.c:34]: Name this?

**Review by nanaian — CHANGES_REQUESTED** [review]
Good cod
Should rename those funcs per the comment

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀 This PR matches 7 functions (+0.07%) on `us`.

✅ This PR fixes 1 warnings!

**Pull request merged**